service-base/src/main/java/org/eclipse/hono/service/management/credentials/CommonCredential.java

```diff
@@ -55,37 +55,67 @@ public String getAuthId() {
         return authId;
     }
 
-    public void setAuthId(final String authId) {
+    /**
+     * Set the authId for this credential.
```

Comment:
I appreciate the effort you took for adding JavaDoc to the setters. Can you also please use an *active* verb at the beginning of the first sentence? In this case it would be
> Sets the identifier that the device uses for authentication to protocol adapters.

It is much more helpful to the reader if the description explains the semantics of what the method does instead of just rephrasing the method and parameter name. Maybe you can rewrite the other JavaDoc comments accordingly, WDYT?

Reply:
yes true.. will update the comments